docs(plugins/acme) add doc for dummy route setup and staging cleaning (#1843)
Kong/docs.konghq.com: app/_hub/kong-inc/acme/index.md

      required: false
      default: "`https://acme-v02.api.letsencrypt.org`"
      description: |

>         The ACMEv2 API endpoint to use, user might use [Let's Encrypt staging environemnt](https://letsencrypt.org/docs/staging-environment/) during testing. Kong doesn't automatically delete staging certificates, if you use same domain to test and use in production, you will need to delete those certificates manaully after test.

Reviewer: ```suggestion
        The ACMEv2 API endpoint to use. You can use the [Let's Encrypt staging environment](https://letsencrypt.org/docs/staging-environment/) during testing. Note that Kong doesn't automatically delete staging certificates: if you use same domain to test and use in production, you will need to delete those certificates manually after testing.
```

Ubuntu/Debian and `/etc/ssl/certs/ca-bundle.crt` for CentOS/Fedora/RHEL.

#### Enable the Plugin

> For all the domains that you need to get certificate, make sure `DOMAIN/.well-known/acme-challenge`

Reviewer: It sounds like you need to map this for each domain - if that's right, then use this phrasing:

```suggestion
For each the domain that needs a certificate, make sure `DOMAIN/.well-known/acme-challenge`
```

is mapped to a Route in Kong. You can check this by sending

> `curl KONG_IP/.well-known/acme-challenge/x -H "host:DOMAIN"` and expect a response `Not found`.

Reviewer: ```suggestion
`curl KONG_IP/.well-known/acme-challenge/x -H "host:DOMAIN"` and getting the response `Not found`.
```

If not, add a Route and a dummy Service to catch this route.
```bash
# add a dummy service if needed
$ curl http://localhost:8001/service \
        -d name=acme-dummy \
        -d url=http://127.0.0.1:65535

> # add a dummy route if needed

Reviewer: Can you add a blank line right before this for consistency with the "add the plugin" section?

Author: Added, thank you!
